aten/src/ATen/native/native_functions.yaml

```diff
@@ -35,6 +35,9 @@
 - func: set_data(Tensor(a!) self, Tensor new_data) -> void
   variants: method
 
+- func: data(Tensor self) -> Tensor
+  variants: method
+
```

Comment:
To fix the CI errors, we likely need to add `data` to `SKIP_PYTHON_BINDINGS` in `tools/autograd/gen_python_functions.py`.